runhouse/resources/provenance.py

```diff
@@ -10,15 +10,14 @@
 from runhouse.constants import LOGS_DIR
 from runhouse.globals import configs, rns_client
 from runhouse.logger import get_logger
-from runhouse.resources.blobs import file
 
 # Need to alias so it doesn't conflict with the folder property
 from runhouse.resources.folders import Folder, folder as folder_factory
 from runhouse.resources.hardware import _current_cluster, _get_cluster_from, Cluster
 from runhouse.resources.resource import Resource
 from runhouse.rns.top_level_rns_fns import resolve_rns_path
 from runhouse.rns.utils.api import log_timestamp, resolve_absolute_path
-from runhouse.utils import StreamTee
+from runhouse.utils import StreamTee, write_data_to_a_file
```

Comment:
actually i think if we're killing provenance soon we don't need to bother with this refactor - if `write_data_to_a_file` isn't used anywhere else we can prob remove also

Reply:
might be easiest to add this PR to the stack for removing `provenance` instead of against main